example/plugin/main.go

```diff
@@ -0,0 +1,105 @@
+package main
+
+/*
+ * SPDX-FileCopyrightText: 2023 Siemens AG
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ *
+ * Author: Michael Adler <michael.adler@siemens.com>
+ */
+
+import (
+	"log"
+	"net/http"
+	"os"
+	"strings"
+	"time"
+
+	"github.com/siemens/wfx/generated/plugin"
+	"github.com/siemens/wfx/middleware/plugin/ioutil"
+)
+
+const queueSize = 64
+
+type QueueEntry struct {
+	start   time.Time
+	request *plugin.RequestT
+}
+
+func main() {
+	file, err := os.OpenFile("plugin.log", os.O_APPEND|os.O_CREATE|os.O_WRONLY, 0o644)
```

Comment:
It's probably worth mentioning what the working directory of a launched plugin is, the directory in which the executable is?

Reply:
Yes good point, I've added a comment